### PR TITLE
12.90-13.10 update

### PR DESCRIPTION
**Pull request by EPuncker**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

**Issues addressed:** 

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Comment by Zbizu** [comment]
missing fencer outfit

I'm not sure which looktypes it has

**Comment by Zbizu** [comment]
points of interest for the update:
- podium packet changed
- items with timers/charges need 5 extra bytes to display
- npc trade allows 100 stacks/items per transaction now (don't copy from me, it's unfinished), amount in buy/sell is u16 now.

File: data/scripts/actions/others/taming.lua [hunk 1]
```diff
@@ -359,6 +359,14 @@ local config = {
 			{sound = "GRRRRRRRRR", text = "The noble lion majestically rejects your amulet."}
 		},
 		success = {sound = "Grrr", text = "The noble lion will now accompany you as a friend and ally."}
+	},
+	[] = { -- colourful water lily
+		name = "giant beaver",
+		id = ,
```
Comment (ramon-bernardo) [inline data/scripts/actions/others/taming.lua:365]: id?

**Comment by EPuncker** [comment]
closing this as there was no interest in the c++ code part

**Comment by natanFernandesDev** [comment]
I've been trying to get this to work, but my client always crashes, do you have any additional tips that might help me?

**Comment by EPuncker** [comment]
> I've been trying to get this to work, but my client always crashes, do you have any additional tips that might help me?

it is missing A LOT of source changes to make it playable without crash

**Pull request closed without merging**